setup.cfg

```diff
@@ -0,0 +1,24 @@
+[metadata]
+name = markermap
+version = 0.0.1
+author = Wilson Gregory, Beel Sarwar
+author_email = wilson.g.gregory@gmail.com
```

Comment:
Let me know if you want your name to be included in the author list, and also if you want your email there. There can only be one author_email I believe if we are going to put it on pip, but we can add emails in the author line like `Wilson Gregory <yadayada@foo.com>`. We could also make a separate email and just give everyone access to it.